main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -70,6 +70,11 @@ class Flix {
     "Div.flix" -> LocalResource.get("/src/library/Div.flix"),
     "Rem.flix" -> LocalResource.get("/src/library/Rem.flix"),
     "Exp.flix" -> LocalResource.get("/src/library/Exp.flix"),
+    "And.flix" -> LocalResource.get("/src/library/And.flix"),
+    "Or.flix" -> LocalResource.get("/src/library/Or.flix"),
+    "Xor.flix" -> LocalResource.get("/src/library/Xor.flix"),
+    "Shl.flix" -> LocalResource.get("/src/library/Shl.flix"),
+    "Shr.flix" -> LocalResource.get("/src/library/Shr.flix"),
 
```

Comment:
I would like to rename these to:
- BitAnd
- BitOr
- BitXor
- BitShl
- BitShr

(or could be Bitwise- thoughts?) 

I would not rename the operations themselves (yet).

I just think that having a TC called `And` is too general.

Reply:
I agree.